src/Features/CSharp/Portable/UsePatternMatching/CSharpAsAndNullCheckCodeFixProvider.cs

```diff
@@ -89,18 +89,23 @@ void RemoveStatement(StatementSyntax statement)
             var asExpressionLocation = diagnostic.AdditionalLocations[2];
 
             var declarator = (VariableDeclaratorSyntax)declaratorLocation.FindNode(cancellationToken);
-            var comparison = (BinaryExpressionSyntax)comparisonLocation.FindNode(cancellationToken);
+            var comparison = (ExpressionSyntax)comparisonLocation.FindNode(cancellationToken);
             var asExpression = (BinaryExpressionSyntax)asExpressionLocation.FindNode(cancellationToken);
+
+            var rightSideOfComparison =
+                comparison is BinaryExpressionSyntax binaryExpression
+                ? (SyntaxNode)binaryExpression.Right
+                : ((IsPatternExpressionSyntax)comparison).Pattern;
```

Comment:
for consistency with the rest of the code in the IDE, could you please write this either as:

```c#
            var rightSideOfComparison = comparison is BinaryExpressionSyntax binaryExpression
                ? (SyntaxNode)binaryExpression.Right
                : ((IsPatternExpressionSyntax)comparison).Pattern;

// or as

            var rightSideOfComparison =
                comparison is BinaryExpressionSyntax binaryExpression
                    ? (SyntaxNode)binaryExpression.Right
                    : ((IsPatternExpressionSyntax)comparison).Pattern;
```

Having deeper wrapping at the same nesting makes this pretty hard to read.

Thanks~